rpm/tarantool-expirationd.spec

```diff
@@ -29,6 +31,10 @@ make test
 %install
 install -d %{buildroot}%{_datarootdir}/tarantool/
 install -m 0644 expirationd.lua %{buildroot}%{_datarootdir}/tarantool/
+curl -L https://tarantool.io/avEfjSi/release/1.10/installer.sh | bash
+yum install -y tarantool
+tarantoolctl rocks install luatest
+CMAKE_DUMMY_WEBUI=YES tarantoolctl rocks install cartridge
```

Comment:
I doubt that integration tests should be run as part of package building process.

We can either skip them or setup a CI flow that will install the module from the package and run full testing suite.

Reply:
new tests will run in gh ci